src/adapters/mod.rs

```diff
@@ -66,8 +66,10 @@ impl<T: Controller> AdapterManager<T> {
         let scripts_path = &self.controller.get_profile().path_for("thinkerbell_scripts.sqlite");
         ThinkerbellAdapter::init(manager, scripts_path).unwrap(); // FIXME: no unwrap!
         let profile_openzwave = &self.controller.get_profile().path_for("openzwave");
-        let openzwave_device = self.controller.clone().get_config().get("openzwave", "device");
-        OpenzwaveAdapter::init(manager, profile_openzwave, openzwave_device).unwrap();
+
+        // device is for backwards compatability
```

Comment:
wondering if we need to bother about backward compatibility (BTW: it's `backward compatibility` :) ) now or if a mail on the ML would be enough :)